### ISSUE TITLE
Stopped container disappears from ps -a

### ISSUE THREAD

**Issue by corrieb** [vmware/vic#1385]
Easy to reproduce on ESX:

Not sure what's going on here, but this will be in the common path of our users.
```
vagrant@ubuntu-1604-vmware:~/vic/src/github.com/vmware/vic/bin$ docker -H 10.118.69.143:2376 --tls ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
3219dcecb2bd        busybox             "date"              7 minutes ago       Stopped                                 ig_heisenberg
vagrant@ubuntu-1604-vmware:~/vic/src/github.com/vmware/vic/bin$ docker -H 10.118.69.143:2376 --tls create busybox top
0473213dbb565e06fce44045a21d3531c0e62f16fe93f23b4df01d2f3adc2802
vagrant@ubuntu-1604-vmware:~/vic/src/github.com/vmware/vic/bin$ docker -H 10.118.69.143:2376 --tls start 0473213dbb565e06fce44045a21d3531c0e62f16fe93f23b4df01d2f3adc2802
0473213dbb565e06fce44045a21d3531c0e62f16fe93f23b4df01d2f3adc2802
vagrant@ubuntu-1604-vmware:~/vic/src/github.com/vmware/vic/bin$ docker -H 10.118.69.143:2376 --tls ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED                  STATUS              PORTS               NAMES
3219dcecb2bd        busybox             "date"              8 minutes ago            Stopped                                 ig_heisenberg
0473213dbb56        busybox             "top"               Less than a second ago   Running                                 lever_hamilton
vagrant@ubuntu-1604-vmware:~/vic/src/github.com/vmware/vic/bin$ docker -H 10.118.69.143:2376 --tls stop 0473213dbb565e06fce44045a21d3531c0e62f16fe93f23b4df01d2f3adc2802
0473213dbb565e06fce44045a21d3531c0e62f16fe93f23b4df01d2f3adc2802
vagrant@ubuntu-1604-vmware:~/vic/src/github.com/vmware/vic/bin$ docker -H 10.118.69.143:2376 --tls ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
3219dcecb2bd        busybox             "date"              8 minutes ago       Stopped                                 ig_heisenberg
```

**Comment by mhagen-vmware** [comment]
Assigning myself to make sure a test gets written for this, if I haven't already.

**Comment by mdubya66** [comment]
making TP3. assigning to @matthewavery.

**Comment by cgtexmex** [comment]
This is a dupe of #1337

**Comment by corrieb** [comment]
@cgtexmex want me to close this or close the other one?

**Comment by cgtexmex** [comment]
Probably -- the other gives the reason why it's happening...I've looked at it a bit, but haven't had time to figure it out....

**Comment by corrieb** [comment]
@cgtexmex They do seem related, for sure. Given we have @mhagen-vmware following for test purposes and a target on this one, I suggest we keep it open

**Comment by hickeng** [comment]
#1337 is closed and complete - leaving this open for writing of a regression test.

**Comment by mhagen-vmware** [comment]
Fixed by #1549
